AuxTel/Troubleshooting/ATCS/AuxTel-Dome-Fails-to-Arrive-in-Position.rst

```diff
@@ -0,0 +1,187 @@
+      The transition must be quick enough so the dome shutter does not start closing and the recovery is faster; 
+      if it does close, the next step should deal with the shutter opening again.  
+      ``FAULT`` → ``STANDBY`` → ``START`` → ``DISABLED`` → ``ENABLED``.
+   
+      a. If ATPtg faulted, transition it back to enabled from ASummaryState. 
+
+   2. Select `auxtel/prepare_for/onsky` and move it to the beginning of the ATQueue. 
+      Execute this script to ensure the system is back and ready for observations, with the mirror cover and dome shutter opened, and the dome tracking enabled.
+
+
+   3. Keep ATQueue running to the next target. 
+      Confirmin LOVE that the dome is moving and following the mount.
+
+
+B. **Alternative procedure in case A. fails - Recovery using LOVE and a notebook.**
+
+   1. From the ASummaryState view on LOVE, transition the ATPtg back to enabled following the usual path: ``FAULT`` → ``STANDBY`` → ``START`` → ``DISABLED`` → ``ENABLED``
+   
+   2. Using an instantiated atcs class from a notebook (e.g. the daytime_checkout notebook), ensure that dome following is disabled and perform a dome slew (using a value for az that is at least 15 degrees away from where you are currently pointing) before reactivating `dome_following` by running the following commands:
+
+.. Using an instantiated ATCS class from a notebook (e.g., the daytime_checkout notebook), ensure that dome following is disabled. 
+.. Perform a dome slew using a value for *az* that is at least 15 degrees away from where you are currently pointing. 
+.. Reactivate dome_following by running the following commands:
+
+      .. prompt:: bash
+
+        await atcs.disable_dome_following()
+        dome_az = await atcs.rem.atdome.tel_position.next(flush=True,timeout=10)
+        await atcs.slew_dome_to(az=dome_az.azimuthPosition+15)
+        await atcs.slew_dome_to(az=dome_az.azimuthPosition-15)
+        await atcs.enable_dome_following()
+
+
+
+   3. The dome following and positioning should now be recovered. 
+      From the same notebook, perform a test slew choosing *az*, *el*, and *rot* values that are near your current position to ensure the dome tracks and arrives in the desired position.
+
+
+      .. prompt:: bash
+
+        current_position = atcs.rem.atptg.tel_mountPositions.get()
+        start_az = current_position.azimuthCalculatedAngle[0]
+        start_el = current_position.elevationCalculatedAngle[0]
+        coord=atcs.radec_from_azel(az=start_az+10, el=start_el-10)
+        await atcs.slew_icrs(coord.ra, coord.dec, rot=start_rot, stop_before_slew=False, rot_type=RotType.PhysicalSky)
+
+.. _AuxTel-Dome-Fails-to-Arrive-in-Position-Additional-Information:
+
+Additional Information/Details
+==============================
+
+The default timeout value for a slew triggered from a notebook is very long, so it may not be feasible to wait for it to timeout and you should instead interrupt the execution of the cell using the stop. 
+
```

Comment:
From the troubleshooting template I miss the Post-Condition and the Contingency sections.

Reply:
I agree with Karla. would be good, finished the document with Post-Condition and the Contingency sections.